templates/certified/index.html

```diff
@@ -41,31 +42,20 @@ <h1>Ubuntu certified hardware</h1>
     </div>
   </form>
 </section>
-
 <section class="p-strip">
   <div class="row u-equal-height">
-    <div class="col-6 p-card">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/21efb2ec-desktop.svg",
-        alt="",
-        width="71",
-        height="48",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-      <h2 class="p-card__title p-heading--3">Ubuntu certified desktops</h2>
-      <p class="p-card__content u-sv2">Many of the world's biggest PC manufacturers certify their desktops for Ubuntu, ensuring it always runs as smoothly as its millions of users expect. Ubuntu is fast, reliable, packed with great software and free of viruses. That’s why you’ll find Ubuntu preloaded on desktops across government, education and enterprise, and in homes around the world.</p>
-      <hr />
-      <p class="u-text--muted u-no-margin--bottom">Vendor</p>
+   <div class="col-8">
+      <h2>Ubuntu certified desktops</h2>
+      <p>Many of the world's biggest PC manufacturers certify their desktops for Ubuntu, ensuring it always runs as smoothly as its millions of users expect. Ubuntu is fast, reliable, packed with great software and free of viruses. That’s why you’ll find Ubuntu preloaded on desktops across government, education and enterprise, and in homes around the world.</p>
```

Comment:
This is different in the copy doc:
```suggestion
      <p>Many of the world's biggest PC manufacturers certify their desktops for Ubuntu, ensuring these systems always run as smoothly as their millions of users expect. Ubuntu is fast, reliable, packed with great software, and free of viruses. That’s why you’ll find Ubuntu preloaded on desktops across government, education, enterprise, and in homes around the world..</p>
```